.github/actions/build-pr-site/action.yml

```diff
@@ -3,8 +3,12 @@ description: 'Build site from a PR to check if all xref references are valid'
 
 inputs:
   build-preview-command:
-    description: 'The documentation `build-preview` command to build the preview'
+    description: 'The documentation `build-preview` command to build the preview and validate references. Mandatory when `component-name` is not set. Otherwise, ignored.'
```

Comment:
So this field is not required anymore, true ? If yes, we can put the required key to `false`